gammapy/time/lightcurve.py

```diff
@@ -68,3 +69,35 @@ def simulate_example(cls):
         lc['FLUX_ERR'] = Quantity([0.1, 0.4, 0.7, 0.9], 'cm^-2 s^-1')
 
         return lc
+
+    def compute_fvar(self):
+        """Calculate the fractional excess variance (Fvar) of `LightCurve`.
+
+        """
+        time_min = self['TIME_MIN']
+        time_max = self['TIME_MAX']
```

Comment:
Is this the right place to get autopep8 ?
https://github.com/tell-k/vim-autopep8